frontend/src/features/properties/components/forms/subforms/BuildingForm.tsx

```diff
@@ -202,7 +202,7 @@ const BuildingForm = (props: BuildingProps & FormikProps<any>) => {
           </Col>
           <Col md={6}>
             <Form.Row>
-              <Form.Label>Rentable Area</Form.Label>
+              <Form.Label>Net Rentable Area</Form.Label>
```

Comment:
@jdtoombs I don't think this change will stick around unless you add it to the new Building Form.

Reply:
You'll need to do it again essentially for the `alpha-dev` branch